packages/api/src/routers/auth/google_auth.ts

```diff
@@ -57,16 +57,22 @@ export const validateGoogleUser = async (
   }
 }
 
-const googleMobileClient = new OAuth2Client(env.google.auth.iosClientId)
-const googleWebClient = new OAuth2Client(env.google.auth.clientId)
+const iosClientId = env.google.auth.iosClientId
+const webClientId = env.google.auth.clientId
+
+const googleWebClient = new OAuth2Client(webClientId)
+const googleIOSClient = new OAuth2Client(iosClientId)
 
 export async function decodeGoogleToken(
-  idToken: string
+  idToken: string,
+  isAndroid: boolean
```

Comment:
Instead of passing `isAndroid` in here, it might be more clear to have the calling code pass the clientId. Something like:

```
router.post('/sign-in', async (req, res) => {
    const { token, provider } = req.body
    const payload = await createMobileSignInResponse(token, provider)
    const { token, provider, source } = req.body
    const payload = await createMobileSignInResponse(clientIdForSource(source), token, provider)
```